src/coreclr/src/debug/daccess/request.cpp

```diff
@@ -4106,23 +4128,58 @@ HRESULT ClrDataAccess::DACTryGetComWrappersObjectFromCCW(CLRDATA_ADDRESS ccwPtr,
 #endif
 
     if (qiAddress != GetEEFuncEntryPoint(ManagedObjectWrapper_QueryInterface))
+    {
+        return FALSE;
+    }
+
+    return TRUE;
+}
+
+TADDR ClrDataAccess::DACGetManagedObjectWrapperFromCCW(CLRDATA_ADDRESS ccwPtr)
+{
+    if (!DACIsComWrappersCCW(ccwPtr))
+    {
+        return NULL;
+    }
+
+    ULONG32 bytesRead = 0;
+    TADDR managedObjectWrapperPtrPtr = ccwPtr & InteropLib::ABI::DispatchThisPtrMask;
+    TADDR managedObjectWrapperPtr;
```

Comment:
Nit:  Better to assign this to 0 defensively.  A poorly written ReadVirtual implementation might not assign it under a bad circumstance, better to have `nullptr` there instead of uninitialized memory.

Reply:
Thanks!